packages/grafana-ui/src/components/Divider/Divider.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta, Story, Preview, ArgTypes } from '@storybook/blocks';
+import { Divider } from './Divider';
+
+<Meta title="MDX|Divider" component={Divider} />
+
+# Divider
+
+Used to create separation between large sections of page content. Or used to separate smaller sections
+of components. i.e in the info section of the header.
```

Comment:
I would like to see some of the points from [this list](https://github.com/grafana/grafana/blob/main/contribute/style-guides/storybook.md#mdx-file-structure) here. What are the use cases? When would I want to use a vertical separator? When should I not use it?

Reply:
We could add this [section](https://docs.google.com/document/d/1qU5E4na1ARhAdtU9yfVHwgYlDLprauEfNYjz6Y6FawY/edit#heading=h.lcg0m1snay6g) from the docs Catherine wrote about "when not to use". At the bottom of that doc there are also some examples where some of these would be useful right now, but I'm not sure about adding "live examples" on storybook.

But also in general there should be a different task for this `Divider` component that is adding documentation to it, which involves adding it to the Saga website and possibly also updating the one in storybook, although this balance of what is in Saga website and what is in these docs is still something to be determined...

Reply:
Thank you both for feedback, I have updated the documentation.